recipes/libwebp/webp.inc

```diff
@@ -0,0 +1,17 @@
+
+SRC_URI = "https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-${PV}.tar.gz"
+
+inherit auto-package-libs auto-package-utils autotools
+
```

Comment:
you'll have to disable mingw build in this, using the COMPATIBLE_HOST_ARCH dance